Core/Goals/PullTargetGoal.cs

```diff
@@ -39,7 +39,8 @@ public PullTargetGoal(ILogger logger, ConfigurableInput input, PlayerReader play
             AddPrecondition(GoapKey.incombat, false);
             AddPrecondition(GoapKey.hastarget, true);
             AddPrecondition(GoapKey.pulled, false);
-            AddPrecondition(GoapKey.withinpullrange, true);
```

Comment:
How can you be sure, you **can** pull the target if its not required to be in the `withinpullrange` ?

Reply:
Just use "I" btn and you'll be always in range.